packages/pigeon/lib/ast.dart

```diff
@@ -17,7 +17,8 @@ class Node {}
 /// Represents a method on an [Api].
 class Method extends Node {
   /// Parametric constructor for [Method].
-  Method({this.name, this.returnType, this.argType, this.isAsynchronous});
+  Method(
```

Comment:
(fyi, here and in other places, when the dart formatter does ugly breaks like this (where there's just a random newline after an open bracket and then the rest is indented with the closing bracket like on the next line) you can usually convince the formatter to give slightly better results by putting a trailing comma before the close bracket.)